WcaOnRails/app/controllers/registrations_controller.rb

```diff
@@ -134,12 +141,17 @@ def update_all
         RegistrationsMailer.notify_registrant_of_deleted_registration(registration).deliver_now
       end
       flash.now[:warning] = "#{"Registration".pluralize(registrations.length)} deleted"
+    when "export-selected"
     else
       raise "Unrecognized action #{params[:registrations_action]}"
     end
 
     respond_to do |format|
-      format.js { }
+      if params[:registrations_action] == "export-selected"
+        format.js { render "redirect_to_export" }
```

Comment:
nitpick: I feel like most of our calls to render actually use a symbol rather than a string.